docs/bbob-largescale/functions/source/index.rst

```diff
@@ -290,36 +289,40 @@ and :math:`\sum_{i=1}^{n_b}s_i = n`. In this case, the matrices
 is also an orthogonal matrix.
 
 .. Dimo: such a matrix will not exist in all dimensions, right? What for example if :math:`n` is prime? We should be more careful in the definition here (e.g. restricting the potential dimensions or allowing :math:`B_{n_b}` to be smaller than :math:`s_ \times s_i`).
+.. Wassim: I don’t see how :math:`n` being a prime would be a problem. Up to this point, we only require the sum of the block-sizes to be equal to :math:`n`; later, we will define the values of these block-sizes and then, I agree, we should mention that the last block can be, in theory, smaller (all dimensions larger than 40 are multiples of 40 in our case)
```

Comment:
Okay, I read the part above again and indeed it is not specified yet, that all blocks have the same size. Both comments can therefore be deleted before accepting the pull-request since this is mentioned below as proposed.

Reply:
Done.